data/sysusers.d/eos-kolibri.conf.in

```diff
@@ -1 +1 @@
-u      @KOLIBRI_USER@ -    "Kolibri"  @KOLIBRI_USER_HOME@
+u      @KOLIBRI_USER@ 132:139   "Kolibri"  @KOLIBRI_USER_HOME@
```

Comment:
These should be options in `meson_options.txt` and set in this file by `meson.build`, like the other fields here (username, homedir).

The defaults in `meson_options.txt` can be the numbers used in EOS.